01.1-codeofconduct.Rmd

```diff
@@ -0,0 +1,78 @@
+
+By participating in this community, you agree not to engage in behavior contrary to these values at any Bioconductor-sponsored event (in person or virtual, including but not limited to talks, workshops, poster sessions, social activities) or electronic communication channel (including but not limited to community-bioc Slack, the support site, online forums, package review site and social media communications). Furthermore, we require all participants to have identifiable accounts in Bioconductor online forums. Accounts that do not adhere to this after request to de-anonymise may be deleted.
+
+We do not tolerate harassment, intimidation, or bullying of community members. Sexual language and imagery are not appropriate in presentations, communications or in online venues, including chats.
+
+Any person/s violating the Code of Conduct may be sanctioned or expelled temporarily or permanently from an electronic platform or event at the discretion of the Code of Conduct committee.
+Examples of unacceptable harassment, intimidation, and bullying behavior
+
+Harassment includes, but is not limited to:
+
+    Making comments in chats, to an audience or personally, that belittle or demean another person
+    Sharing sexual images online
+    Harassing photography or recording
+    Sustained disruption of talks or other events
+    Unwelcome sexual attention
+    Advocating for, or encouraging, any of the above behavior
+
+Intimidation and bullying include, but are not limited to:
+
+    Aggressive or browbeating behavior
+    Mocking or insulting another person’s intellect, work, perspective, or question/comment
+    Making reference to someone’s gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion, or other personal attributes in the context of a scientific discussion
+    Deliberately making someone feel unwelcome
+    Trolling behaviour (deliberately inflammatory or offensive messages)
+    Sustained off-topic posts
+
+**Enforcement**
+
+Anyone asked to stop harassing or intimidating behavior are expected to comply immediately.
+
+If a person/s contravene the Code of Conduct the Code of Conduct committee retains the right to take any action that ensures a welcoming environment for all community members. This includes warning the alleged offender or temporary/permanent expulsion from the event and/or electronic platforms under Bioconductor’s control.
+
+The Code of Conduct committee may take action to redress anything designed to, or with the clear impact of, disrupting an event or electronic communication platform or making the environment hostile for any community member.
+
+We expect everyone in the Bioconductor community to comply with the Code of Conduct when participating in Bioconductor events and online communication platforms.
+Reporting
```

Comment:
This seems to be a subtitle in the original text in 
https://bioconductor.github.io/bioc_coc_multilingual/

Can you check the original text, and make the appearance similar.. (titles are titles, bolding is bolding etc).

Reply:
This also